lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -404,6 +410,10 @@ function REPL( options ) {
 	* @private
 	*/
 	function onClose() {
+		// Disable bracketed-paste mode in TTY environments:
+		if ( self._isTTY ) {
```

Comment:
Would it make sense to make this configurable? E.g., `disableBracketedPasteOnExit` (perhaps a better name, such as `autoDisableBracketedPaste` or something else)? E.g., suppose my terminal session already supports bracketed paste before I type `make repl`. Currently, when I later exit the REPL, bracketed paste gets disabled, and I'd need to re-enable bracketed paste in my terminal session. If we make it configurable, a user could at least toggle whether to auto-disable and preserve their previous behavior.

Reply:
Makes sense. Do you mean a setting for this?

Reply:
Yes, exactly.

Reply:
Am I understanding you correctly that you want to explicitly prompt a user before exiting regarding whether to preserve bracketed paste?

Reply:
Yeah, I think we can skip an exit prompt for now. This would be straightforward to add if a user ever requests this feature. A setting should be enough, especially once we are able to persist and load REPL configurations.